jupyter-covid19/Dockerfile

```diff
@@ -57,6 +57,39 @@ RUN touch /home/$NB_USER/covid19-notebook/PFB_example.ipynb
 ADD --chown=jovyan:users https://raw.githubusercontent.com/uc-cdis/covid19-tools/$COVID_TOOLS_BRANCH/covid19-notebooks/IL_tab_charts.ipynb /home/$NB_USER/covid19-notebook/
 RUN touch /home/$NB_USER/covid19-notebook/IL_tab_charts.ipynb
 
+ADD --chown=jovyan:users https://raw.githubusercontent.com/uc-cdis/covid19-tools/$COVID_TOOLS_BRANCH/covid19-notebooks/3D_Protein_Vis/3D_Protein_Vis.ipynb /home/$NB_USER/covid19-notebook/
+RUN touch /home/$NB_USER/covid19-notebook/3D_Protein_Vis.ipynb
+
+ADD --chown=jovyan:users https://raw.githubusercontent.com/uc-cdis/covid19-tools/$COVID_TOOLS_BRANCH/covid19-notebooks/3D_Protein_Vis/3D_Protein_Vis_7D4F_gui.png /home/$NB_USER/covid19-notebook/
+RUN touch /home/$NB_USER/covid19-notebook/3D_Protein_Vis_7D4F_gui.png
+
+ADD --chown=jovyan:users https://raw.githubusercontent.com/uc-cdis/covid19-tools/$COVID_TOOLS_BRANCH/covid19-notebooks/3D_Protein_Vis/3D_Protein_Vis_7D4F_view.html /home/$NB_USER/covid19-notebook/
+RUN touch /home/$NB_USER/covid19-notebook/3D_Protein_Vis_7D4F_view.html
+
+ADD --chown=jovyan:users https://raw.githubusercontent.com/uc-cdis/covid19-tools/$COVID_TOOLS_BRANCH/covid19-notebooks/3D_Protein_Vis/3D_Protein_Vis_A_view.html /home/$NB_USER/covid19-notebook/
+RUN touch /home/$NB_USER/covid19-notebook/3D_Protein_Vis_A_view.html
```

Comment:
do we need the PNG and HTML files in the workspace?

Reply:
Yes, we need those png and html files, they are included in the Jupyter notebooks for illustration purpose. Sometimes we inserted a flowchart or an example of the expected outputs within the Jupyter notebooks. If we don't copy them to the workspace, they won't show up in the notebooks. @paulineribeyre